CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ Updated Motoko to [0.10.2](https://github.com/dfinity/motoko/releases/tag/0.10.2
 
 ### Frontend canister
 
+Fixed an issue where the `allow_raw_access` configuration value was being affected by the order of  declarations in the configuration file in `.ic-assets.json`. 
```

Comment:
In the changelog, please describe the issue in enough detail that someone with a project, the behavior of which will change after this fix, will be able to recognize that fact.